src/components/PersonalPage/PersonalPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {Card, Image, ListGroup} from 'react-bootstrap';
+
+import './PersonalPage.css';
+import backgroundImageStyling from '../../styles/backgroundImageStyling';
+import MainNavbar from "../../utils/MainNavbar";
+import DevCard from "./DevCard";
+import SponsorCard from "./SponsorCard";
+
+//Require any images
+const logo = require('../../images/swarmLogoIcon.png');
+
+//Get random background image
+const background = backgroundImageStyling();
+
+class PersonalPage extends React.Component<any, any> {
+    
+    componentDidMount() {
+        // @ts-ignore, object could possibly be null
+        document.getElementsByTagName("BODY")[0].classList.add('pageBody');
+
+        setTimeout(function (){
+            // @ts-ignore, object could possibly be null
+            document.getElementById('cardHolder').classList.add('fade-in');
+        }, 1)
+    }
+    
```

Comment:
If the DevCard and the SponsorCard are going to be child components of the personal page card then in the fild structure it might be best for the two card elements to get their own folders within this PersonalPage folder or have them seperated altogether.